Gem/Code/Source/Components/UI/UiSettingsComponent.cpp

```diff
@@ -83,20 +67,11 @@ namespace MultiplayerSample
 
     void UiSettingsComponent::Activate()
     {
-        // Initialize our user settings 
-
-        // Initialize the current streaming image mip bias setting.
-        if (AZ::IConsole* console = AZ::Interface<AZ::IConsole>::Get(); console)
-        {
-            int16_t mipBias = 0;
-            console->GetCvarValue("r_streamingImageMipBias", mipBias);
-            m_settings.m_streamingImageMipBias = aznumeric_cast<uint8_t>(mipBias);
-        }
-
-        // Initialize the graphics API type
-        m_settings.m_atomApiType = AZ::RHI::Factory::Get().GetAPIUniqueIndex();
-
-        // There's currently no way to initialize the master volume, this doesn't seem to be fetchable anywhere.
+        // Make sure to load and apply the current user settings when this component activates.
```

Comment:
Comment seems to say developer needs to take action, but then says component will reapply the desired settings.

So is this more "Loads and applies the current user setting when this component..."?